scripts/foglamp

```diff
@@ -164,6 +164,21 @@ foglamp_reset() {
   # Remove user data: scripts
   rm -rf "$FOGLAMP_DATA/scripts"
   echo "Removed user data in $FOGLAMP_DATA/scripts"
+  # Remove user data: logs
+  find "$FOGLAMP_DATA/logs" -depth -type f -exec rm {} \;
+  echo "Removed user data in $FOGLAMP_DATA/logs"
+  # Remove user data: var
+  find "$FOGLAMP_DATA/var" -depth -type f -exec rm {} \;
+  echo "Removed user data in $FOGLAMP_DATA/var"
+  # Remove user data: extras, with one exclusion
+  find "$FOGLAMP_DATA/extras" -depth -type f -not -name fogbench_sensor_coap.template.json -exec rm {} \;
+  echo "Removed user data in $FOGLAMP_DATA/extras"
+  # Remove user etc: extras, with one exclusion
+  find "$FOGLAMP_DATA/etc" -depth -type f -not -name storage.json -exec rm {} \;
```

Comment:
We need to keep default `etc/certs/` which are 
foglamp.key, foglamp.cert, 
ca.key, ca.cert, ca.csr,  user.key, user.cert, 
admin.key & admin.cert (created by scripts/certificates & scripts/auth_certificates)  but delete others.
Also it appears to me we need to retain  `etc/kerberos` directory structure with its README.rst but its contents i.e. .keytab files should be removed.